import/realm-config/generic/catenax-central/CX-Central-realm.json

```
@@ -1975,6 +1975,28 @@
           "clientRole": true,
           "containerId": "114605ea-9c64-4dff-9bc7-90fe02a004c3",
           "attributes": {}
+        },
+        {
+          "id": "9f8c4c27-2a5e-4d5c-9a6b-8e7d4c0e7b3a",
+          "name": "Connector Management",
+          "description": "Allowed to access connector registration features without using UI",
+          "composite": true,
+          "composites": {
+            "client": {
+              "Cl2-CX-Portal": [
+                  "view_tech_user_management",
+                  "add_tech_user_management",
+                  "technical_roles_management",
+                  "view_connectors",
+                  "add_connectors",
+                  "modify_connectors",
+                  "delete_connectors"
```

Are all those roles really needed?

I remember that the requestor of https://github.com/eclipse-tractusx/sig-release/issues/1186 mentioned that the [Offer Management role](https://github.com/eclipse-tractusx/portal-iam/blob/6aa6c9723d3aa7d9741b6a52df73aca7decabed4/import/realm-config/generic/catenax-central/CX-Central-realm.json#L1670) (minus the app specific roles) does have sufficient permissions for what they wanted to achieve, only issue was that the Offer Management role isn't meant for active participants companies.

I'd at least assume that the roles for the technical user management aren't needed:

```suggestion
                  "view_connectors",
                  "add_connectors",
                  "modify_connectors",
                  "delete_connectors"
```

---

> Hi @evegufy, thanks for the review.
> 
> User story expects this feature here as far as I understand: "As a active participant I want to be able to create technical users to register connectors via API call" and it made sense to me since technical user creation/selection is also part of connector registration via UI.
> 
> Maybe @FloBeer can comment?

Hi @typecastcloud ,

yes, first step is to be able to create such kind of technical user as active participant with the respective roles. Second step would be to leverage the technical user credentials for API call to create/modify connectors etc.